hedera-node/configuration/testnet/application.properties

```diff
@@ -4,7 +4,6 @@
 # It does not have to match bootstrap.properties once that file is deprecated by the switch from mono-service to hedera-config.
 
 #Overrides that differ based on the network
-ledger.id=0x01
-contracts.chainId=296
 bootstrap.genesisPublicKey=e06b22e0966108fa5d63fc6ae53f9824319b891cd4d6050dbf2b242be7e13344
-contracts.knownBlockHash=
+contracts.chainId=296
+ledger.id=0x01
```

Comment:
`contracts.knownBlockHash` is not used anymore. Reordered the other properties to match file `0.0.121` on testnet.

Reply:
Should we issue this update immediately after this is merged?

Reply:
No, we need to wait until 0.48 was deployed. I have not checked the code of 0.47.